pkg/artifactory/resource_artifactory_permission_target.go

```diff
@@ -400,6 +405,10 @@ func resourcePermissionTargetExists(d *schema.ResourceData, m interface{}) (bool
 	if _, ok := d.GetOk("repositories"); ok {
 		_, resp, err := c.V1.Security.GetPermissionTargets(context.Background(), d.Id())
 
+		if resp == nil {
```

Comment:
Check `err` first. If you check error first you probably never need to check if `resp` is `nil`.